src/execution/common.rs

```diff
@@ -47,49 +46,52 @@ pub(crate) fn call_module<T: WasmerEnv>(
         Ok(v) => v,
         Err(_) => abi_bail!("negative amount of coins in Call"),
     };
+    let env = ctx.data().clone();
     let bytecode = env.get_interface().init_call(address, raw_coins)?;
     let module = get_module(&*env.get_interface(), &bytecode)?;
 
     let remaining_gas = if cfg!(feature = "gas_calibration") {
         Ok(u64::MAX)
     } else {
-        get_remaining_points(env)
+        get_remaining_points(&env, ctx)
```

Comment:
Same as in `as_abi.rs` is that possible to use only ctx ?

Reply:
same as above